frontend/javascripts/oxalis/model/accessors/tool_accessor.ts

```diff
@@ -138,6 +148,9 @@ function _getDisabledInfoForProofreadTool(
     } else {
       explanation = disabledSkeletonExplanation;
     }
+  } else {
+    explanation =
+      "A mapping that does not support proofreading actions is pinned to this annotation.";
```

Comment:
```suggestion
      "A mapping that does not support proofreading actions is pinned to this annotation. Most likely, the annotation layer was modified before (e.g. by brushing).";
```

Just a suggestion to make this more clear to users.